### ISSUE TITLE
Simple SshClient.RunCommand gets no output

### ISSUE THREAD

**Issue by ghost** [sshnet/SSH.NET#535]
```csharp
SshClient client = new SshClient(address.ToString(), port, username, password);
            client.Connect();
            var command = client.RunCommand("sleep 5;ls /");
            string output;
            using (var stream = command.ExtendedOutputStream)
            using (StreamReader reader = new StreamReader(stream))
            {
                output = reader.ReadToEnd();
            }

            Console.WriteLine(output);
```

output is always empty - should I have to create a Shell in this case to see the output of the command? If I connect to my box and check ps while the command is executing (hence `sleep` above) I can see that the command is running successfully.

**Comment by ghost** [comment]
`var command = client.CreateCommand("ls /").Execute();` works as expected, but it'd be better to have access to the Output and Error streams in the long term

**Comment by pitu1** [comment]
Hi! I ran into the same problem, but I really need to access the result. Did you find the solution?

**Comment by FlukeFan** [comment]
> Hi! I ran into the same problem, but I really need to access the result. Did you find the solution?

Nope.  I think I just redirected the output on the remote server so I could inspect it later.

**Comment by Rob-Hague** [comment]
The code should have been using `command.OutputStream` not `ExtendedOutputStream`, which is typically just stderr
